binary.go

```diff
@@ -66,7 +66,7 @@ func (bs *Binaries) Add(c []byte) *Binary {
 }
 
 // GetContent returns a string which is the plaintext content of a binary
-func (b Binary) GetContent() (string, error) {
+func (b Binary) GetContent() ([]byte, error) {
```

Comment:
Maybe just to weigh in here on this change:
This would be a breaking change and require a major version update (again..)

Reply:
Oh, sorry. I didn’t though this change will be added to the merge request.